### DISCUSSION TITLE
Leaderboard scores sometimes show as Standardised when Classic is selected

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17021] [Q&A] [answered]
Occasionally when scrolling through Leaderboards, some scores will show up as Standardised. Clicking on them to bring up the Result Screen will correctly show it as Classic and also stay like that when going back to the Leaderboards.

![](https://pek.li/z1albd.png)
![](https://pek.li/n3kpc1.png)

Couldn't find a way to consistently repro this, it just happens seemingly at random. (Reloading the Leaderboards, by toggling selected mods for example, will eventually show this issue)

**Comment by bdach** [comment]
potentially related to / duplicate of https://github.com/ppy/osu/discussions/16934

**Comment by smoogipoo** [comment]
Does this still happen on the most recent build? Can't repro here :(

**Reply by respektive** [reply]
Still happens on latest release for me.

**Reply by smoogipoo** [reply]
Does it always happen on the same maps? Can you link one of these maps?

**Reply by respektive** [reply]
Not sure if it's always happening on the same maps, but here is a video of me randomly going through maps and links to the maps where it happened on.

https://user-images.githubusercontent.com/22381687/160747085-a672542e-bcf0-4021-8565-d12d468eabe3.mp4

https://osu.ppy.sh/beatmapsets/1282842#osu/2664322
https://osu.ppy.sh/beatmapsets/780908#osu/1640253
https://osu.ppy.sh/beatmapsets/391671#osu/853094
https://osu.ppy.sh/beatmapsets/944723#osu/1972913

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/17549
